### ISSUE TITLE
Fact column filter

### ISSUE THREAD

**Issue by ofmorescale** [jjmontesl/cubesviewer#79]
Hi!

I wanted to do a filter on the fact column and didn't find yet how to do it.

I have, for example, a "user" dimension, and a fact that measures the "number of accesses".
When I drilldown the user dimension, I can see the number of accesses per user.
But, now I want to see the table or graph drilldowned by users, but only where the number of accesses per user is more than 50 (for example). That is, I want to do a filter on the fact column under an arbitrary drilldown.

Is it possible to do this automatically? (without having to filter the dimension column by hand, which is practically intractable given the number of users)

Thanks and hope my question is clear enough!

**Comment by jjmontesl** [comment]
For this I'd create another dimension, for users, where their number of accesses added as a discretized dimension (ie. <10, 10-50, 50-100, 100-200, 200-500, >500). 

This way you can have cube views over your user metrics including one or more of those groups. This is a pretty common technique (having measures also as discretized dimensions).

I hope that helps.

**Comment by ofmorescale** [comment]
Yes, thanks for your response.
We implemented that solution in some cases, and in others we simply listed the 20 more common email domains (@domain.com) (we wanted to do a drilldown by email domain), and directly restricted the data that was inserted in the cubesviewer database.

I don't know much about how cubesviewer handles the filters, but as we know that the cubes server is supposed to do all the possible calculations (and it does, because the data is available), we don't wanted to start doing those calculations in our backend. 

But probably having measures as categorized dimensiones is the best solution for now.
Thanks!!
